### PR TITLE
Manual build fixes

### PR DESCRIPTION
**Pull request by cjakeman**
https://bugs.launchpad.net/or/+bug/1889789
Fixed some bad references in the Manual which stop links from working plus other build errors which are red in the console.

### CODE REVIEW

File: Source/Documentation/Manual/physics.rst [hunk 1]
```diff
@@ -1176,14 +1176,11 @@ OR models the heat loss from a boiler with some standard default settings,
 however the model can be customised to suit the locomotive by adjusting the 
 following settings.
 
--``ORTSBoilerSurfaceArea`` ==> Surface area of the boiler / fire box that impacts 
-heat loss. Default UoM - ft^2
+- ``ORTSBoilerSurfaceArea`` - Surface area of the boiler / fire box that impacts heat loss. Default UoM - ft^2
```
Comment (YoRyan) [inline Source/Documentation/Manual/physics.rst:1179]:
```suggestion
- ``ORTSBoilerSurfaceArea`` - Surface area of the boiler / fire box that impacts heat loss. Default UoM - ft\ :sup:`2`
```

File: Source/Documentation/Manual/physics.rst [hunk 2]
```diff
@@ -2846,7 +2844,7 @@ where
 
 .. math::
 
-    A=\frac{0.00003318\cdot\rho\cdot F_t}{(1-F_{tr}/F_t)^2},
+    A=\frac{0.00003318\cdot rho\cdot F_t}{(1-F_{tr}/F_t)^2},
```
Comment (YoRyan) [inline Source/Documentation/Manual/physics.rst:2847]:
```suggestion
    A=\frac{0.00003318\cdot\rho\cdot F_t}{(1-F_{tr}/F_t)^2},
```
I suggest undoing this. `\rho`, or ρ, is the correct symbol for the density of air.

File: Source/Documentation/Manual/physics.rst [hunk 3]
```diff
@@ -2855,7 +2853,7 @@ where
 +-----------------------------------------------------------+-----------------------------------------------------------+
 |F\ :sub:`t` -- tunnel cross-sectional area (m\ :sup:`2`\ ) |F\ :sub:`tr` -- train cross-sectional area (m\ :sup:`2`\ ) |
 +-----------------------------------------------------------+-----------------------------------------------------------+
-||rgr| -- density of air ( = 1.2 kg/m\ :sup:`3`\ )          |R\ :sub:`t` -- tunnel perimeter (m)                        |
+|\rho -- density of air ( = 1.2 kg/m\ :sup:`3`\ )           |R\ :sub:`t` -- tunnel perimeter (m)                        |
```
Comment (YoRyan) [inline Source/Documentation/Manual/physics.rst:2856]:
```suggestion
|:math:`\rho` -- density of air ( = 1.2 kg/m\ :sup:`3`\ )   |R\ :sub:`t` -- tunnel perimeter (m)                        |
```
Use the ρ symbol here, too.

File: Source/Documentation/Manual/physics.rst (hunk 2 above)
Comment (cjakeman) [reply Source/Documentation/Manual/physics.rst]: Hi Ryan,

Thanks for this. I couldn't see how to do it.

Please approve,

Chris

**Pull request merged**